README.md

```diff
@@ -49,11 +49,12 @@ Body:
 ``` json
 {
     "callbackUrl": "string representing absolute url",
-    "metadata": { "any": "valid json object" }
+    "metadata": { "any": "valid json object" },
+    "maxSizePerItem": "5MB"
```

Comment:
attribute name is incorrect, should be constraints.maxSizePerItem according to ticket